src/common/dav/wal_tx.c

```diff
@@ -482,6 +485,7 @@ dav_wal_replay_cb(uint64_t tx_id, struct umem_action *act, void *base)
 			act->ac_op_bits.addr / PAGESIZE, act->ac_op_bits.addr % PAGESIZE,
 			act->ac_op_bits.pos, act->ac_op_bits.num);
 		off = act->ac_op_bits.addr;
+		size = sizeof(uint64_t);
```

Comment:
This set/clear bits only supports sizeof(uint64_t)? It's not from this PR, but I don't understand the "ASSERT_rt((pos >=0 && (pos +,num) <= 64) below.

Reply:
Oh, never mind.